js/src/redux/providers/signerReducer.js

```diff
@@ -62,7 +62,7 @@ export default handleActions({
   signerSuccessConfirmRequest (state, action) {
     const { id, txHash } = action.payload;
     const confirmed = Object.assign(
-      state.pending.find(p => p.id === id),
+      state.pending.find(p => p.id === id) || { id },
```

Comment:
Think it solves that issue. Still unsure exactly why it happens, it should have all the info. (Just a nightmare to replicate - I have tried.)